### DISCUSSION TITLE
Add sensor.solar_energy_today to Energy Dashboard

### DISCUSSION THREAD

**Discussion by swa72** [emsesp/EMS-ESP32#1062] [General]
The stock configuration does not list `sensor.solar_energy_today` in HA's new energy dashboard. Adding

```
sensor.solar_energy_today:
  device_class: energy
  state_class: total_increasing
```

to `customize.yaml` did the trick for me.

**Comment by proddy** [comment]
This is on the todo list - see https://github.com/emsesp/EMS-ESP32/issues/129

**Comment by proddy** [comment]
merged with #129
